src/Model/Payment/AbstractPayment.php

```diff
@@ -123,7 +127,7 @@ public function isModuleActive(): bool
      */
     public function isEnabled(): bool
     {
-        return (bool) $this->scopeConfig->getConfigByPath(static::XML_CONFIG_ENABLED)->getValue();
+        return static::XML_CONFIG_ENABLED !== '' && (bool) $this->scopeConfig->getConfigByPath(static::XML_CONFIG_ENABLED)->getValue();
```

Comment:
can u actually check this above and throw an exception if it's not set? Should be the logic for all constants that are required to be set by the developer. This way this will be more intuitive to implement.